split-webapp/split/src/pages/Split.js

```diff
@@ -32,6 +32,17 @@ class Split extends Component {
     const name = document.getElementById("name").value;
     document.getElementById("name").value = "";
 
+    if (name.length === 0) {
+      alert("Please enter a valid non-empty name.");
```

Comment:
Using alerts is probably not the cleanest way to do the validation but I think it's fine, the next group can make it look nicer if they want :)